src/Parser/RestoDruid/Modules/Traits/ArmorOfTheAncients.js

```diff
@@ -0,0 +1,12 @@
+
+import Module from 'Parser/Core/Module';
+
+/**
+ * Armor of the Ancients
+ * Ironbark reduces all damage taken by the target by an additional 2%.
+ */
+class ArmorOfTheAncients extends Module {
+//TODO
```

Comment:
We can probably just do the DRPS calculation here. For the particular case of Ironbark, DRPS is going to be pretty comparable to HPS. Maybe include in tooltip the explanation that it's DRPS and also says average extra damage reduced per Ironbark cast.